packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -5,19 +5,24 @@
 
 const nunjucks = require('nunjucks');
 
-// todo:
-// test
-// refactor
-// jsdoc
-
 class Question {
 	title;
 	question;
 	description;
 	type;
 	fieldName;
+	validators = [];
+	hint;
+	bulletPoints = [];
+	details = {
+		title: '',
```

Comment:
Just sense checking my understanding  - if we evaluated if(details.title) with an empty string like so it would return false so we can use that to do conditional rendering of the nun jucks templates on the server https://developer.mozilla.org/en-US/docs/Glossary/Truthy?